CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ### Unreleased
 
+### 1.1.0
+- Release to facilitate a Travis Pypi deployment.
```

Comment:
I don't see the value in this. The source code in PyPi is what is in GitHub, there were no code injection/change attack vectors that we are aware of. If we are releasing a new version here, then the same logic should apply to Runner et. al., and they should also be version bumped?

Reply:
Resolved offline